src/tests/category/S25_category_filter.test.js

```diff
@@ -0,0 +1,47 @@
+import { group } from 'k6';
+import OptionsUtil from '../../utils/options.util';
+import { createMetrics } from '../../utils/metric.util';
+import EnvironmentUtil from '../../utils/environment.util';
+import { CategoryFixture } from '../../fixtures/category.fixture';
+import CategoryPage from '../../pages/yves/category.page';
+
+const testConfiguration = {
+  ...EnvironmentUtil.getDefaultTestConfiguration(),
+  id: 'S25',
+  group: 'Category',
+  metrics: ['S25_get_category'],
+  thresholds: {
+    S25_get_category: {
+      smoke: ['avg<300'],
+      load: ['avg<600'],
+    },
+  },
+};
+
+const { metrics, metricThresholds } = createMetrics(testConfiguration);
+export const options = OptionsUtil.loadOptions(testConfiguration, metricThresholds);
+
+export function setup() {
+  const dynamicFixture = new CategoryFixture({
+    categoryCount: 1,
+    productCount: 100,
+  });
+
+  return dynamicFixture.getData();
+}
+
+export default function (data) {
+  const category = CategoryFixture.iterateData(data);
+
+  group('Category Filter Products', () => {
+    const categoryPage = new CategoryPage();
+    const response = categoryPage.get(category.url, {
```

Comment:
it's a bit strange, lets pass this params to fixtures instead getting it from fixtures